src/main/java/org/terasology/dynamicCities/region/RegionEntityManager.java

```diff
@@ -184,36 +176,46 @@ public boolean cellIsLoaded(Vector2i position) {
 
     public List<EntityRef> getRegionsInCell(EntityRef region) {
         LocationComponent regionLocation = region.getComponent(LocationComponent.class);
-        Vector2i pos = new Vector2i(regionLocation.getLocalPosition().x(), regionLocation.getLocalPosition().z());
+        Vector3fc loc = regionLocation.getLocalPosition();
+        Vector2i pos = new Vector2i(loc.x(), loc.z(), RoundingMode.FLOOR);
         return  getRegionsInCell(pos);
     }
 
     public boolean checkSidesLoadedLong(Vector2i pos) {
```

Comment:
This is only called from `checkSidesLoadedLong(EntityRef)`. I think both can be removed.